packages/st2bundle/install_rhel6.md

```diff
@@ -0,0 +1,48 @@
+## Install IUS repo
+sudo yum install -y https://dl.iuscommunity.org/pub/ius/stable/Redhat/6/x86_64/ius-release-1.0-14.ius.el6.noarch.rpm
+
```

Comment:
Should be then same for el6/el7 `wget -qO - "https://setup.ius.io/" | sudo bash -s`.
To make it more generic.